query08.sql

```diff
@@ -1,6 +1,16 @@
 /*
-  Give the five most popular stations between 7am and 10am in 2019.
+  Give the five most popular starting stations between 7am and 10am in 2019.
 */
 
 -- Enter your SQL query here
-select ...
+with q2_starts as (
+	SELECT start_time, start_station, EXTRACT(HOUR FROM start_time) as start_hour FROM public.indego_trips_2019_q2
+)
+select start_station, count(*)
+from q2_starts
+where start_hour between 11 and 14
```

Comment:
I'm pretty sure the times are reported in Eastern time, not UTC, so no time zone conversion needed. But otherwise seems correct.